apps/app/src/app/(shell)/create-agent/CreateAgentForm.tsx

```
@@ -0,0 +1,177 @@
+'use client';
+
+import React, { useState } from "react";
+import { useForm } from "react-hook-form";
+import { Loader2Icon } from "lucide-react";
+import { SupportedModel } from "~/config/models";
+import { useCreateAgent } from "~/lib/data/agents";
+import { queryClient } from '~/lib/data/queryClient';
```

ESLint

[error] 'queryClient' is defined but never used.

(unused-imports/no-unused-imports)